apps/addons/buttons.py

```diff
@@ -233,11 +234,13 @@ def __init__(self, text, url, os=None, file=None):
 
 # Cache it for a year.
 @cache_page(60 * 60 * 24 * 365)
+@non_atomic_requests
```

Comment:
This looks right to me, the [docs](https://docs.djangoproject.com/en/1.6/topics/db/transactions/#django.db.transaction.non_atomic_requests) also say:

> It only works if it’s applied to the view itself.